<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotepadClass</class>
 <widget class="QMainWindow" name="NotepadClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>637</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="TabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"> background-color: rgb(239, 239, 239);

</string>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>637</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionSave_all"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
    <addaction name="actionClose_all"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionClear"/>
    <addaction name="actionSelect_all"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_in"/>
    <addaction name="actionZoom_out"/>
   </widget>
   <widget class="QMenu" name="menuFind">
    <property name="title">
     <string>Find</string>
    </property>
    <addaction name="actionFind_in_file"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionRestore_default"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuFind"/>
   <addaction name="menuSettings"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewFile"/>
   <addaction name="actionopenFile"/>
   <addaction name="actionsaveFile"/>
   <addaction name="actionsaveAll"/>
   <addaction name="actioncloseFile"/>
   <addaction name="actioncloseAll"/>
   <addaction name="separator"/>
   <addaction name="actioncut"/>
   <addaction name="actioncopy"/>
   <addaction name="actionpaste"/>
   <addaction name="separator"/>
   <addaction name="actionundo"/>
   <addaction name="actionredo"/>
   <addaction name="separator"/>
   <addaction name="actionzoomIn"/>
   <addaction name="actionzoomOut"/>
   <addaction name="separator"/>
   <addaction name="actionfind"/>
   <addaction name="actionfont"/>
   <addaction name="actiontextColor"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNewFile">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/newFile.png</normaloff>:/Notepad/Resources/newFile.png</iconset>
   </property>
   <property name="text">
    <string>NewFile</string>
   </property>
   <property name="toolTip">
    <string>New</string>
   </property>
  </action>
  <action name="actionopenFile">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/openFile.png</normaloff>:/Notepad/Resources/openFile.png</iconset>
   </property>
   <property name="text">
    <string>openFile</string>
   </property>
   <property name="toolTip">
    <string>Open..</string>
   </property>
  </action>
  <action name="actionsaveFile">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/saveFile.png</normaloff>:/Notepad/Resources/saveFile.png</iconset>
   </property>
   <property name="text">
    <string>saveFile</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionsaveAll">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/saveAll.png</normaloff>:/Notepad/Resources/saveAll.png</iconset>
   </property>
   <property name="text">
    <string>saveAll</string>
   </property>
   <property name="toolTip">
    <string>Save all</string>
   </property>
  </action>
  <action name="actioncloseFile">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/closeFile.png</normaloff>:/Notepad/Resources/closeFile.png</iconset>
   </property>
   <property name="text">
    <string>closeFile</string>
   </property>
   <property name="toolTip">
    <string>Close</string>
   </property>
  </action>
  <action name="actioncloseAll">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/closeAll.png</normaloff>:/Notepad/Resources/closeAll.png</iconset>
   </property>
   <property name="text">
    <string>closeAll</string>
   </property>
   <property name="toolTip">
    <string>Close all</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+Åš</string>
   </property>
  </action>
  <action name="actionSave_all">
   <property name="text">
    <string>Save all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionClose_all">
   <property name="text">
    <string>Close all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+W</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/cut.png</normaloff>:/Notepad/Resources/cut.png</iconset>
   </property>
   <property name="text">
    <string>cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/copy.png</normaloff>:/Notepad/Resources/copy.png</iconset>
   </property>
   <property name="text">
    <string>copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/paste.png</normaloff>:/Notepad/Resources/paste.png</iconset>
   </property>
   <property name="text">
    <string>paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/undo.png</normaloff>:/Notepad/Resources/undo.png</iconset>
   </property>
   <property name="text">
    <string>undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionredo">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/redo.png</normaloff>:/Notepad/Resources/redo.png</iconset>
   </property>
   <property name="text">
    <string>redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionzoomIn">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/zoomIn.png</normaloff>:/Notepad/Resources/zoomIn.png</iconset>
   </property>
   <property name="text">
    <string>zoomIn</string>
   </property>
   <property name="toolTip">
    <string>Zoom in</string>
   </property>
  </action>
  <action name="actionzoomOut">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/zoomOut.png</normaloff>:/Notepad/Resources/zoomOut.png</iconset>
   </property>
   <property name="text">
    <string>zoomOut</string>
   </property>
   <property name="toolTip">
    <string>Zoom out</string>
   </property>
  </action>
  <action name="actionfind">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/find.png</normaloff>:/Notepad/Resources/find.png</iconset>
   </property>
   <property name="text">
    <string>find</string>
   </property>
   <property name="toolTip">
    <string>Find</string>
   </property>
  </action>
  <action name="actionfont">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/font.png</normaloff>:/Notepad/Resources/font.png</iconset>
   </property>
   <property name="text">
    <string>font</string>
   </property>
   <property name="toolTip">
    <string>Font</string>
   </property>
  </action>
  <action name="actiontextColor">
   <property name="icon">
    <iconset resource="Notepad.qrc">
     <normaloff>:/Notepad/Resources/blackSquare.png</normaloff>:/Notepad/Resources/blackSquare.png</iconset>
   </property>
   <property name="text">
    <string>textColor</string>
   </property>
   <property name="toolTip">
    <string>Text color</string>
   </property>
  </action>
  <action name="actionZoom_in">
   <property name="text">
    <string>Zoom in</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+=</string>
   </property>
  </action>
  <action name="actionZoom_out">
   <property name="text">
    <string>Zoom out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+_</string>
   </property>
  </action>
  <action name="actionFind_in_file">
   <property name="text">
    <string>Find in file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionRestore_default">
   <property name="text">
    <string>Restore default</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header location="global">TabWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Notepad.qrc"/>
 </resources>
 <connections/>
</ui>
